toolkit/tools/pkg/imagecustomizerlib/cosicommon.go

```diff
@@ -287,3 +287,10 @@ func populateVerityMetadata(source string, verity *Verity) error {
 
 	return nil
 }
+
+func getArchitecture() string {
```

Comment:
nit: Rename to `getArchitectureForCosi` since all the functions in the same directory are in the same namespace.